website/content/en/docs/tutorials/iperf.md

```diff
@@ -164,23 +164,78 @@ deployment "cl-dataplane" successfully rolled out
 
 ## Enable cross-cluster access
 
-In this step, we enable connectivity access between the iPerf3 client and server.
+In this step, we enable connectivity access between the iPerf3 client and server.<br>
+For each step, you have an example demonstrating how to apply the command from a file or providing the complete CRD associated with the command.
+### Peers setup
 
-1. First, add the peers to each cluster:
+Add the peers to each cluster:
 
-    Client cluster:
+Client cluster:
 
-    ```sh
-    export SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' server-control-plane`
-    curl -s $IPERF3_FILES/clusterlink/peer-server.yaml | envsubst | kubectl apply -f -
-    ```
+{{< tabpane text=true >}}
+{{% tab header="File" %}}
 
-    Server cluster:
+```sh
+export SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' server-control-plane`
+curl -s $IPERF3_FILES/clusterlink/peer-server.yaml | envsubst | kubectl apply -f -
+```
+
```

Comment:
nit: be consistent in adding/using empty lines between shortcodes and content. I think it is better without:
```
{{< tabpane text=true >}}
{{% tab header="File" %}}
... text ...
{{% /tab %}}
{{% tab header="Full CRD" %}}
...
{{% /tab %}}
{{% /tabpane %}}
```
Note applies to next tabpane as well